article/_posts/2021-07-29-DiSCo.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: blog
+title: "The Diffusion-Simulated Connectivity (DiSCo) Challenge"
+tags: tractography tractometry dMRI
+author: "Antoine Théberge"
+pdf: "http://hardi.epfl.ch/static/events/2021_challenge/"
+---
+
+# Introduction
+The Diffusion-Simulated Connectivity (DiSCo) Challenge is a connectivity estimation challenge presented at CDMRI, a workshop organized at MICCAI 2021. The goal of the challenge is to estimate the connectivity of a synthetic phantom composed of 16 regions of interest (ROI) connected by interlacing fiber bundles. The participating teams must provide their own connectivity matrix and the team with the highest correlation coefficient with the ground-truth matrix will be declared winner.
+
+# Motivations
+
+The estimation of brain structural connectivity from diffusion-weighted magnetic resonance (DW-MR) images  is still an open subject and depends on many steps. Roughly, connectivity estimation from a set of DW-MR images acquisitions goes as follows:
+
+- Image denoising and resampling
+- Signal processing and modelling
+- Tractography
+- Tractogram filtering and post-processing
+- Connectivity estimation
+
+Because each step has its own set of methods and each methods has its own set of parameters, it is primordial to evaluate the performance of the pipelines currently used.
+
+# Data
+
+![](/article/images/disco/fig1.jpg)
```

Comment:
This image doest not seem to be included in the pull request

Reply:
Woops indeed, fixed in c50a3c81507212f26a998f2e07a298c29ea7d25e